tools/finish_release.py

```diff
@@ -125,12 +129,17 @@ def create_github_release(github_access_token, tempdir, version):
                                      release_notes,
                                      draft=True)
 
+    # SSH into CSS and sign executable, then retrieve
+    username = getpass.getuser()
+    host = css
+    css_path = username + '@' + host + ':~/signed-exes/certbot-beta-installer-win32-signed.exe'
+    subprocess.run(["scp", css_path , tempdir + '/windows-installer/'])
```

Comment:
Feel free to share the instructions privately out of band, but I don't entirely understand how this process works.

What are the steps involved for signing into the code signing server signing the Windows installer? How is the correct Windows executable to sign selected and how do we prevent an old one accidentally from being used here?

Reply:
I have script on the CSS and will share out of band. 👍🏾